engine/classes/Elgg/Config.php

```diff
@@ -95,6 +95,15 @@
  * @property bool          $remove_branding 						Is Elgg branding disabled
  * @property int           $remove_unvalidated_users_days			The number of days after which unvalidated users will be removed
  * @property bool          $require_admin_validation
+ * @property string        $securitytxt_acknowledgments             Security.txt link to acknowledgments
```

Comment:
variable name should have each word separated by an underscore... `$security_txt_*`

Reply:
done